department-of-veterans-affairs/vets-api
# [35634] Adding inherited proofing callback after login.gov authentica…

…tion success

## Description of change
This PR adds a callback intended to be called at the end of the inherited proofing process by login.gov. The user must be authenticated on va.gov for this callback, and it will add a record to prove inherited proofing has occurred, will log the user out of the MHV authentication, and will redirect them to a login.gov authentication

## Original issue(s)
department-of-veterans-affairs/va.gov-team#35634

## Things to know about this PR
- To test, log in with any credential provider
- Inspect the results of the `users` route call, expectation is `inherited_proof_verified` should be equal to `false` 
- Navigate to `localhost:3000/inherited_proofing/callback`
- Expected behavior is you should now be logged out and redirected to the login.gov authentication page
- If you log back in with your original account, inspect the results of the `users` route call, expectation is `inherited_proof_verified` should now be equal to `true`
